service/lombok.config

```diff
@@ -0,0 +1,2 @@
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
```

Comment:
Should it be commited? Or even version controlled?

Reply:
added lombok.config to .gitignore

Reply:
This still needs to be removed from pr